.github/workflows/approve-regression-tests-command.yml

```diff
@@ -60,9 +63,32 @@ jobs:
             [1]: ${{ steps.job-vars.outputs.run-url }}
 
       - name: Approve regression tests
-        id: approve
+        id: approve-regression-tests
```

Comment:
I've the impression we have three different context values:
* `Regression Tests on ${{ github.event.inputs.connector-name }}`
* `Regression Test Results Reviewed and Approved`
* `Regression tests` (set in the `subcommand` of `regression_tests.yml`

Can you share how these three articulate, I'm not sure I get why we need 3 different ones.